src/nvim/option.c

```diff
@@ -908,19 +908,11 @@ static vimoption_T
    SCRIPTID_INIT},
   {"iminsert",    "imi",  P_NUM|P_VI_DEF,
    (char_u *)&p_iminsert, PV_IMI,
-#ifdef B_IMODE_IM
```

Comment:
Do you know what this is? I'm struggling to understand what this does... something related to input methods I guess...